x-pack/plugins/security_solution/server/endpoint/services/actions/constants.ts

```diff
@@ -5,7 +5,19 @@
  * 2.0.
  */
 
+import { AGENT_ACTIONS_INDEX, AGENT_ACTIONS_RESULTS_INDEX } from '@kbn/fleet-plugin/common';
+import {
+  ENDPOINT_ACTIONS_INDEX,
+  ENDPOINT_ACTION_RESPONSES_INDEX_PATTERN,
+} from '../../../../common/endpoint/constants';
 /**
  * The Page Size to be used when searching against the Actions indexes (both requests and responses)
```

Comment:
Now that we have a list API, I need to revise this text here. I think its only used for the Log api

Reply:
This is used also used by details and list APIs.